regulations/urls.py

```diff
@@ -165,8 +165,6 @@
 
     # A regulation section without chrome
     # Example: http://.../partial/201-4/2013-10704
-    url(rf'^partial/(?P<label_id>{match_section})/(?P<version>{match_version})$',
-        lt_cache(PartialSectionView.as_view()), name='partial_section_view'),
```

Comment:
Now that these aren't being used, what else can be removed that would have called it? And can `PartialSectionView` and `PartialParagraphView` now be removed?